When we link against clang's builtin library
(LDFLAGS+=" $($CC print-libgcc-file-name)") we get errors like:

    errors.o ../coregrind/libcoregrind-arm-linux.a ../VEX/libvex-arm-linux.a -lgcc ../coregrind/libgcc-sup-arm-linux.a 
    ld: error: duplicate symbol: __aeabi_memcpy
    >>> defined at aeabi_memcpy.S.o:(__aeabi_memcpy) in archive /home/builder/.termux-build/_cache/android-r23b-api-24-v5/lib64/clang/12.0.8/lib/linux/libclang_rt.builtins-arm-android.a
    >>> defined at m_main.c:2602 (/home/builder/.termux-build/valgrind/src/coregrind/m_main.c:2602)
    >>>            libnolto_coregrind_arm_linux_a-m_main.o:(.text+0x5A1C) in archive ../coregrind/libcoregrind-arm-linux.a

Hopefully it is fine to remove these overrides now.

--- a/coregrind/m_main.c
+++ b/coregrind/m_main.c
@@ -2633,7 +2633,7 @@
 #endif /* defined(VGP_arm_linux) */
 
 /* Some Android helpers.  See bug 368529. */
-#if defined(__clang__) \
+#if 0 && defined(__clang__) \
     && (defined(VGPV_arm_linux_android) \
         || defined(VGPV_x86_linux_android) \
         || defined(VGPV_mips32_linux_android) \
